### ISSUE TITLE
"..." more link needs better accessibility

### ISSUE THREAD

**Issue by LJWatson** [tootsuite/mastodon#1953]
The "..." link does not have an accessible name, so it is not recognised by screen readers.
```
<a class="dropdown__trigger icon-button" href="#dropdown-trigger" style="font-size: 18px; width: 18px; line-height: 18px;">
<i class="fa fa-fw fa-ellipsis-h" style="vertical-align: middle;"></i>
</a>
```

Without a visible text label it will be difficult for some people to understand the meaning of the icon (and therefore the purpose of the link).

The link also does not indicate to screen reader users that content will be revealed when the link is activated.

A link is also the wrong type of control to use. Typically a link fetches another resource (like a document or page), whereas a button invokes an action (like submit a form or open a menu).

Suggest using a simple disclosure design pattern:
```
<button aria-expanded="false"><i></i> More</button>
```

When the button is activated and the options are displayed, the `aria-expanded` attribute should be set to `true`. When the options are collapsed again, the `aria-expanded` attribute should return to `false`. This will indicate to screen reader users that the button will expand or collapse some content.

The ```<button>``` element is the appropriate type of interactive element to use. In addition the browser provides the expected keyboard interaction (it's actionable with the space and enter keys).

Including the "More" text inside the button and keeping it visible makes the purpose of the button easy for everyone to understand, and also ensures the button remains usable for people using custom style sheets.
